### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.web.models.validate_response_error_py3.ValidateResponseError
  fullName: azure.mgmt.web.models.validate_response_error_py3
  langs:
  - python
  module: azure.mgmt.web.models.validate_response_error_py3
  name: validate_response_error_py3
  source:
    id: validate_response_error_py3
    path: azure-mgmt-web\azure\mgmt\web\models\validate_response_error_py3.py
    remote:
      branch: master
      path: azure-mgmt-web\azure\mgmt\web\models\validate_response_error_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.web.models.validate_response_error_py3
references:
- fullName: azure.mgmt.web.models.validate_response_error_py3.ValidateResponseError
  isExternal: false
  name: ValidateResponseError
  parent: azure.mgmt.web.models.validate_response_error_py3
  uid: azure.mgmt.web.models.validate_response_error_py3.ValidateResponseError
